magefiles/magefile.go

```diff
@@ -356,7 +358,14 @@ func (Docs) Serve() error {
 
 // Generate generates CLI references
 func (Docs) Generate() error {
-	// TODO
+	ver, errv := version()
+	if errv != nil {
+		return errv
+	}
+	err := doc.GenMarkdownTree(commands.NewApp(ver), "./docs/docs/references/cli")
+	if err != nil {
+		return err
+	}
```

Comment:
I found a flag to disable auto gen tag.

```suggestion
	ver, err := version()
	if err != nil {
		return err
	}
	cmd := commands.NewApp(ver)
	cmd.DisableAutoGenTag = true
	if err = doc.GenMarkdownTree(cmd, "./docs/docs/references/cli"); err != nil {
		return err
	}
	return nil
```